src/System.Collections/tests/Generic/Stack/Stack.Generic.Tests.cs

```diff
@@ -227,7 +215,28 @@ public void Stack_Generic_TrimExcess_AfterClearingAndAddingAllElementsBack(int c
                 Assert.Equal(count, stack.Count);
             }
         }
-
-        #endregion
+        
+        // Contains
+        
+        [Theory]
+        [MemberData(nameof(ValidCollectionSizes))]
+        public void Stack_Generic_Contains_BasicFunctionality(int count)
+        {
+            Stack<T> stack = GenericStackFactory(count);
+            List<T> list = stack.ToList();
+            
+            // Stack should contain all items that result from enumeration
+            Assert.All(list, item => Assert.True(stack.Contains(item)));
+            
+            stack.Clear();
+            
+            // Stack should not contain any items after being cleared
+            Assert.All(list, item => Assert.False(stack.Contains(item)));
+            
+            foreach (T item in list) stack.Push(item);
+            
+            // Stack should contain whatever items are pushed to it
+            Assert.All(list, item => Assert.True(stack.Contains(item)));
+        }
```

Comment:
>If that's not covered elsewhere and is important to test, why only for Stack?

:+1: definitely feel free to add any new Collections tests that test general behavior to the base test classes directly.